map-widget/qml/MapWidget.qml

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0
-import QtQuick 2.5
-import QtLocation 5.3
-import QtPositioning 5.3
+import QtQuick 
+import QtLocation 
+import QtPositioning 
```

Comment:
yeah, I noticed this in many places... the version numbers were needed in the beginning when they implied required functionality - I think we should remove them everywhere in our QML code unless there's a reason to have a version.

Reply:
I know in the past you needed the version numbers, and with Qt6 they prefer it without. This is the most important change to get google maps working.